README.md

```diff
@@ -1,2 +1,57 @@
-# yoti-node-sdk-sandbox
-Yoti Node SDK Sandbox
+# Yoti NodeJS Sandbox SDK
+
+Welcome to the Yoti NodeJS Sandbox. This repo contains the tools you need to test your NodeJS back-end integration.
+
+## Requirements
+
+### Node version
+Please refer to [Travis](https://travis-ci.com/getyoti/yoti-node-sdk-sandbox) to see all compatible Node versions.
+
+## Installing the Sandbox
+
+To import the Yoti SDK inside your project, you can use your favourite dependency management system.
+If you are using NPM, you can use the following command to set the Yoti SDK as a dependency:
+
+```shell
+npm install @getyoti/sdk-sandbox
+```
+
+## Configuration
+
+* `CLIENT_SDK_ID` is the SDK identifier generated by Yoti Hub in the Key tab when you create your app. Note this is not your Application Identifier which is needed by your client-side code.
+
+* `path/to/your-application-pem-file.pem` is the path to the application pem file. It can be downloaded only once from the Keys tab in your Yoti Hub.
+
+Please do not open the pem file as this might corrupt the key and you will need to create a new application.
+
+## Profile Token Creation
+
+```javascript
+const fs = require('fs');
+
+const {
+  SandboxProfileClientBuilder,
+  TokenRequestBuilder,
+} = require('@getyoti/sdk-sandbox');
+
+const CLIENT_SDK_ID = 'your sdk id';
+const PEM = fs.readFileSync(__dirname + '/keys/your-application-pem-file.pem');
+
+const sandboxProfileClient = new SandboxProfileClientBuilder()
+  .withClientSdkId(CLIENT_SDK_ID)
+  .withPemString(PEM)
+  .build();
+
+const tokenRequest = new TokenRequestBuilder()
+  .withFullName(SOME_VALUE)
```

Comment:
Do you think it would be good to add just one more example attribute to this too?

Reply:
I think we could add most/all of them chained together

Reply:
Added more examples in 8170965